pulsar-transformations/src/test/java/com/datastax/oss/pulsar/functions/transforms/ComputeStepTest.java

```diff
@@ -462,6 +497,181 @@ public static Object[] destinationTopicProvider() {
     return new Object[] {"targetTopic", "randomTopic"};
   }
 
+  /**
+   * @return {"old value", "old schema", "old type", "expression", "new schema", "expected value",
+   *     "expectedSchema"}
+   */
+  @DataProvider(name = "primitiveTypeComputeProvider")
+  public static Object[][] primitiveTypeComputeProvider() {
+    return new Object[][] {
+      {
+        "oldValue",
+        Schema.STRING,
+        SchemaType.STRING,
```

Comment:
It seems these values don't really matter.
Maybe we could set `"'value'"` for the expression instead ?